src/pages/Admin/Charity/Deposits/index.tsx

```diff
@@ -0,0 +1,16 @@
+import Content from "../../../Donate/Content";
```

Comment:
should move `Donate/Content` to `components/donation` as it is used now by two pages/features 

then classes that is more related to its external be also factored out and let bounding container provide it 


e.g
in `/donate`
```jsx 
<Content classes="justify-self-center padded-container scroll-mt-6 max-w-[35rem]"/> 
```

and in `admin/deposit` like
```jsx
<Content classes="justify-self-center dark:bg-blue-d6 max-w-lg"/> 
```

also breakpoints be updated from window based  e.g `sm:` to container counterpart `@sm:` as  `donate` is full page while `admin/deposit` is part of the admin layout 

These changes could be done on dedicated PR as right now `donate/{id}` doesn't show up and can't be visually inspected 
<img width="713" alt="image" src="https://user-images.githubusercontent.com/89639563/234176524-390efc7a-c163-415f-b9d8-063932a3f554.png">

Reply:
Thanks @ap-justin! Will start making some of these changes as part of the in-progress work to show fiat-only flow, for when no wallet is connected. You can see the `/donate/{id}` example for ASTs with a published profile. 
A working example: https://rc-v1-8.d2f5hbetruke97.amplifyapp.com/donate/21